src/knownnodes.py

```diff
@@ -60,9 +60,10 @@ def pickle_deserialize_old_knownnodes(source):
     the old format was {Peer:lastseen, ...}
     the new format is {Peer:{"lastseen":i, "rating":f}}
     """
+    global knownNodes
```

Comment:
Please use a comment to suppress pylint warning, like

    global knownNodes # pylint: disable=global-statement